CHANGELOG.md

```diff
@@ -1,13 +1,11 @@
 
-## vNext
+## v0.24.0
 
 - Update Status Lambda - budget_check: Terminate lease if spend > Principal budget amount
+- Add a new `Orphaned` status to accounts (#93)
 - Support `metadata` parameter in `/accounts` API endpoints
 - Add `PUT /accounts/:id` endpoint
 - Fixed bug where child account's DCEPrincipal role trusted itself rather than the master account
-- Add GetUsageByPrincipal
-- Fix default `budget_notification_from_email` TF var (See #143)
-- Replace GetUsageByDateRange with GetUsageByPrincipal method in POST lease validation
```

Comment:
> - Add GetUsageByPrincipal
> - Replace GetUsageByDateRange with GetUsageByPrincipal method in POST lease validation

Thinking these aren't user facing changes, so we don't need them in our changelog

> - Fix default `budget_notification_from_email` TF var (See #143)

This was actually a bug that was only introduced since v0.23.0, so there's no actual change here (ie, it's just "committed a bug, and then fixed the bug")

Reply:
Do we want Users to know that GET usage can be done with various filters?